### PR TITLE
change default values for yogi optimizer

### PR DESCRIPTION
**Pull request by avinashsai**
Hi,
I have implemented yogi optimizer in pytorch and was able to reproduce results from the paper. Here is the repository for the experiments https://github.com/avinashsai/Adaptive-Nonconvex-Optimization/tree/master/PyTorch. This is the implementation according to the details mentioned in the paper.

In this repo, there are couple of changes to be made to coincide with the paper:
1. learning rate is 1e-2 and epsilon is 1e-3.
2. Unlike other optimizers yogi has non-zero initialization for first and second moments. 

I have made changes accordingly.

### CODE REVIEW

File: torch_optimizer/yogi.py [hunk 1]
```diff
@@ -82,9 +90,21 @@ def step(self, closure: OptLossClosure = None) -> OptFloat:
                 if len(state) == 0:
                     state['step'] = 0
                     # Exponential moving average of gradient values
-                    state['exp_avg'] = torch.zeros_like(p)
+                    state['exp_avg'] = nn.init.constant_(
+                            torch.empty_like(
+                                p.data,
+                                memory_format=torch.preserve_format
+                                ),
+                            group["initial_accumulator"]
```
Comment (jettify) [inline torch_optimizer/yogi.py]: Linter complains on double quotes (it enforces consistent quotes in codebase), can you please change it to single ones?

```
flake8 torch_optimizer tests examples setup.py
362torch_optimizer/yogi.py:98:35: Q000 Remove bad quotes
363torch_optimizer/yogi.py:106:35: Q000 Remove bad quotes
364Makefile:5: recipe for target 'flake' failed
365make: *** [flake] Error 1
```
you can run linter locally by doing `make flake`

File: torch_optimizer/yogi.py [hunk 2]
```diff
@@ -9,37 +10,38 @@
...
 class Yogi(Optimizer):
-    r"""Implements Yogi optimization algorithm.
-
-    It has been proposed in `Adaptive Methods for Nonconvex Optimization`__.
-
+    r"""Implements Yogi Optimizer Algorithm.
+    It has been proposed in `Adaptive methods for Nonconvex Optimization`_.
     Arguments:
-        params: iterable of parameters to optimize or dicts defining
+        params (iterable): iterable of parameters to optimize or dicts defining
```
Comment (jettify) [inline torch_optimizer/yogi.py]: No need to add types in doc strings, class constructor is fully types with `mypy` annotations docs builder looking them up there.

**Comment by jettify** [comment]
@avinashsai Thank you! I added few minor tweak requests.

**Comment by avinashsai** [comment]
@jettify I have made changes according to your comments.

**Comment by jettify** [comment]
I will try to push new version to PyPI with your fixes, tonight or tomorrow.

**Comment by jettify** [comment]
New version available on PyPI https://pypi.org/project/torch-optimizer/0.0.1a9/
